.github/workflows/qemu-self-test-seabios.yml

```diff
@@ -0,0 +1,53 @@
+name: Keywords self-tests for Dasharo (coreboot+SeaBIOS) on QEMU Q35
```

Comment:
We should stick to GH's actions. The question is if this would be reliable. Sooner or later, we will exhaust the pool of free actions and storage. I don't see ourselves switching to paid GH, so I would switch to our Woodpecker CI instead.

But I get your point about reusability. My gh-actions-fu was not good enough to be aware of that possibility. There is no space for such work for now, but we should track it as a potential improvement. Based on your comment, I will create an issue for the following PC Engines release.